packages/react/src/ReactNodeViewRenderer.tsx

```diff
@@ -207,6 +217,25 @@ class ReactNodeView extends NodeView<
     this.editor.off('selectionUpdate', this.handleSelectionUpdate)
     this.contentDOMElement = null
   }
+
+  updateElementAttributes() {
+    if (this.options.attrs) {
+      let attrsObj: Record<string, string> = {}
+
+      if (typeof this.options.attrs === 'function') {
+        const extensionAttributes = this.editor.extensionManager.attributes
+        const HTMLAttributes = getRenderedAttributes(this.node, extensionAttributes)
+
+        attrsObj = this.options.attrs({ node: this.node, HTMLAttributes })
+      } else {
+        attrsObj = this.options.attrs
+      }
+
+      Object.keys(attrsObj).forEach(key => {
+        this.renderer.element.setAttribute(key, attrsObj[key])
+      })
```

Comment:
Would prefer to call into the ReactRenderer here so we keep these separated

Reply:
Please review again~ @nperez0111